### PR TITLE
Add EncryptionBoxInput

### PR DESCRIPTION
**Pull request by Keshoid**

### CODE REVIEW

**Review by sertony — APPROVED** [review]

File: EncryptionBoxInput/EncryptionBoxInput.sol [hunk 1]
```diff
@@ -0,0 +1,30 @@
+pragma ton-solidity >= 0.43.0;
+
+interface IEncryptionBoxInput {
```
Comment (sertony) [inline EncryptionBoxInput/EncryptionBoxInput.sol:3]: as @mikhailUshakoff rightly noted that we might want to have the `destroy` method in order to be able to kill the Encryption Box instance from debot once not needed anymore

File: EncryptionBoxInput/README.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+## Functions
+
+`get` - prints prompt message to the user and returns handle to opened encryption box.
+
+arguments:
+
+    answerId: uint32 - id of function callback.
+    prompt: bytes - utf-8 string to print to the user before input, OPTIONAL (empty string is similar to null).
+    algorithm: bytes - utf-8 string with a name of encryption algorithm. Can be one of the following value: "NaclBox",  "NaclSecretBox", "Chacha20" - or other algorithm name supported by Browser.
+    params: TvmCell - packed algorithm parameters.
```
Comment (sertony) [inline EncryptionBoxInput/README.md]: as @mikhailUshakoff noted we are to describe the structure of the params here

File: EncryptionBoxInput/EncryptionBoxInput.sol [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+	function get(uint32 answerId, string prompt, string algorithm, TvmCell params) external returns (uint32 handle);
+    function getSupportedAlgorithms() external returns (string[] names);
+}
+
+library EncryptionBoxInput {
+
+	uint256 constant ID = 0x5b5f76b54d976d72f1ada3063d1af2e5352edaf1ba86b3b311170d4d81056d61;
+	int8 constant DEBOT_WC = -31;
+
+	function get(uint32 answerId, string prompt, string algorithm, TvmCell params) public pure {
```
Comment (mikhailUshakoff) [inline EncryptionBoxInput/EncryptionBoxInput.sol]: How we should pack (TvmCell params) for different algorithm?

File: EncryptionBoxInput/EncryptionBoxInput.sol [hunk 4]
```diff
@@ -0,0 +1,30 @@
+pragma ton-solidity >= 0.43.0;
+
+interface IEncryptionBoxInput {
+
+	function get(uint32 answerId, string prompt, string algorithm, TvmCell params) external returns (uint32 handle);
+    function getSupportedAlgorithms() external returns (string[] names);
```
Comment (mikhailUshakoff) [inline EncryptionBoxInput/EncryptionBoxInput.sol]: add function for box removing
function remove(uint32 handle) external returns (bool res);
to call https://github.com/tonlabs/TON-SDK/blob/master/docs/mod_crypto.md#remove_encryption_box

**Review by mikhailUshakoff — CHANGES_REQUESTED** [review]

File: EncryptionBoxInput/EncryptionBoxInput.sol (hunk 4 above)
Comment (Keshoid) [reply EncryptionBoxInput/EncryptionBoxInput.sol]: Done

File: EncryptionBoxInput/EncryptionBoxInput.sol (hunk 1 above)
Comment (Keshoid) [reply EncryptionBoxInput/EncryptionBoxInput.sol:3]: `remove` method added.

**Review by sertony — APPROVED** [review]

File: EncryptionBoxInput/README.md (hunk 2 above)
Comment (Keshoid) [reply EncryptionBoxInput/README.md]: Deprecated

**Review by sertony — APPROVED** [review]

**Review by mikhailUshakoff — APPROVED** [review]

**Review by BigRoma — APPROVED** [review]

**Pull request merged**